modules/discover/con_benefits-of-rhdh.adoc

```diff
@@ -0,0 +1,38 @@
+:_newdoc-version: 2.18.3
+:_template-generated: 2024-10-29
+
+:_mod-docs-content-type: CONCEPT
+
+[id="benefits-of-rhdh_{context}"]
+= Benefits of {product}
+
+{product} is designed to streamline developer workflow by providing a consistent self-service experience and can simplify application development through pre-built resources and enhanced collaboration tools.
+
+The following are the benefits of {product}:
+
+* Increased developer productivity. Increases productivity by eliminating common organizational challenges, enabling seamless collaboration, and providing clear guidelines for creating, developing, and deploying applications.
+
+* Unified self-service dashboard. Provides development teams with a unified dashboard covering various aspects such as Git, CI/CD, SAST/DAST, Supply Chain, OpenShift/Kubernetes cluster, JIRA, monitoring, API, documentation, and more, facilitated by over 150 plugins. All curated by a platform engineering team, aligning with the company’s best practices.
+
+* Best practices through software templates. Automates organizational best practices by encoding common tasks such as creating new applications, running Ansible jobs, and establishing CI/CD pipelines for production deployment in Git.
+
+* Scalable technical documentation. Code and documentation resides in the same repository, eliminating dependencies on proprietary document systems.
+
+* Efficient onboarding for new developers. New developers can adapt quickly and become productive within a short timeframe.
+
+* Robust enterprise Role-Based Access Control (RBAC). Empowers administrators to create roles, assign users or groups to roles, and implement robust security policies for enhanced access control.
+
+
+[role="_additional-resources"]
+.Additional resources
+* For more information about the different features of {product} and how you can extend the upstream Backstage product by providing additional features such as integration with OpenShift, enterprise role-based access control (RBAC), and dynamic plugins, see link:https://developers.redhat.com/rhdh/overview?intcmp=7015Y000003swEuQAI&source=sso[{product}].
+
+
+= Supported platforms
+
+{product} runs on {ocp-short} {ocp-version-min}-{ocp-version} on AMD64 and Intel 64 (`x86_64`) architectures.
+
+
+[role="_additional-resources"]
+.Additional resources
+For more information about the different supported hosted environments, see link:https://docs.redhat.com/en/documentation/red_hat_openshift_dev_spaces/3.12/html-single/release_notes_and_known_issues/index#supported-platforms[Supported platforms].
```

Comment:
you're linking to the Dev Spaces docs, not the Dev Hub ones. Should be obvious since you're not using a variable for the `{product-version}` but instead hardcoding 3.12 (which is 5 versions old - latest is 3.16). 

Should the link be to this instead?

https://docs.redhat.com/en/documentation/red_hat_developer_hub/1.3/html-single/installing_red_hat_developer_hub_on_openshift_container_platform/index#assembly-install-rhdh-ocp-operator


```suggestion
For more information about the different supported hosted environments, see link:https://docs.redhat.com/en/documentation/red_hat_developer_hub/{product-version}/html-single/installing_red_hat_developer_hub_on_openshift_container_platform/index#assembly-install-rhdh-ocp-operator[Supported platforms].
```